liquigraph-core/src/main/java/org/liquigraph/core/io/xml/LiquigraphEntityResolver.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.io.xml;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.EntityResolver;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+/**
+ * Resolves XML entities against local classpath resources.
+ */
+public class LiquigraphEntityResolver implements EntityResolver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    private static final Map<String, String> knownEntities = new HashMap<>();
+
+    static {
+        knownEntities.put("http://www.liquigraph.org/schema/1.0/liquigraph.xsd", "schema/changelog.xsd");
```

Comment:
that would break as soon as the schema changes

Reply:
I suggest we always include in Liquigraph JAR:
- current schema (www.liquigraph.org/schema/liquigraph.xsd for both HTTP and HTTPS)
- 1.0 schema (www.liquigraph.org/schema/1.0/liquigraph.xsd and fbiville.github.io/liquigraph/schema/1.0/liquigraph.xsd for both HTTP and HTTPS)
- 1.0-RC3 schema (www.liquigraph.org/schema/1.0-RC3/liquigraph.xsd and fbiville.github.io/liquigraph/schema/1.0-RC3/liquigraph.xsd for both HTTP and HTTPS)

so that this resolver can effectively support all existing schemas while preserving both backward and forward compatibilities